tests/src/test/java/alluxio/stress/BatchTaskRunnerIntegrationTest.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.stress;
+
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertEquals;
+
+import alluxio.stress.cli.BatchTaskRunner;
+import alluxio.stress.common.FileSystemClientType;
+import alluxio.stress.master.MasterBenchSummary;
+import alluxio.stress.master.MasterBenchTaskResult;
+import alluxio.util.JsonSerializable;
+
+import com.google.common.collect.ImmutableList;
+import org.junit.Test;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import java.util.List;
+import java.util.Map;
+
+public class BatchTaskRunnerIntegrationTest extends AbstractStressBenchIntegrationTest {
+  @Test
+  public void MasterIntegrationFileTest() throws Exception {
```

Comment:
can you add a few more tests with different parameters? For example the `--client-type` have two options. What about other read/write types? What other combinations make sense?